src/interfaces/IMorpho.sol

```diff
@@ -123,6 +123,8 @@ interface IMorphoBase {
     /// `liquidate` from being used under certain market conditions.
     /// - A very high price returned by the oracle can make the computation of `maxBorrow` in `_isHealthy` overflow, or
     /// the computation of `assetsRepaid` in `liquidate` overflow.
+    /// @dev For a market with less than 1e4 assets borrowed in total, a manipulation is possible to inflate the borrow
+    /// shares' price to the point where borrower can no longer interact with the market.
```

Comment:
```suggestion
    /// @dev The share price of a market with less than 1e4 total borrowed assets can be deflated to a point that borrowing is no longer possible.
```

I am fine with replacing "deflated" with "decreased"

In any case, I don't think that "share price inflation" is the right choice of words: the goal of the attack is to inflate total borrow shares, so deflate the share price

Right?

Reply:
I am also fine with replacing "borrowing is no longer possible" with "borrowing overflows" or "borrowing reverts"

Reply:
Or, to a broader extent, that "can be freely deflated / decreased."

To indicate that the issue is not only about DoSing a market, but also manipulating the share price